src/App.js

```diff
@@ -4,17 +4,22 @@ import AppRouter from './AppRouter';
 import store from './store';
 import Notifier from './components/Notifications/Notifier';
 
+
 // import './App.css';
 
 class App extends Component {
+
     render() {
+
         return (
             <Provider store={store}>
+                
                 <div>
                     {/* <AppNavbar/> */}
                     <Notifier />
                     <AppRouter/>
                 </div>
+
```

Comment:
Remove what seems like unnecessary whitespace and `div`. Do we need better eslint rules?
Can work on that if it is necessary.

Reply:
Yes, you can work on that if you want. While you're at it, try to add react hooks eslint rules as well

Reply:
When I did it was as simple as adding a package. Will investigate some options and try working on it tomorrow night or this week 👍

Reply:
(For reference: https://www.npmjs.com/package/eslint-plugin-react-hooks)

Reply:
Tracked here: https://github.com/NIAEFEUP/nijobs-fe/issues/16

Reply:
PR dropped: #22